src/main/java/storage/Storage.java

```diff
@@ -0,0 +1,39 @@
+package storage;
+
+import exceptions.DukeException;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class Storage {
+    private static final String DEFAULT_PATH = "data.txt";
+    private Path path;
+
+    public Storage() {
+        this.path = Paths.get(DEFAULT_PATH);
+    }
+
+    public List<String> load() throws DukeException {
```

Comment:
I like how your Storage class will create the file if it does not exist rather than throwing the error straight away